test/WebJobs.Script.Tests.Integration/App.config

```diff
@@ -1,112 +1,112 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
```

Comment:
looks like you're introducing whitespace changes somehow? Can these be reverted?